aten/src/ATen/native/cpu/Reduce.h

```diff
@@ -115,41 +115,57 @@ static inline void vectorized_outer_reduction(char** data, int64_t inner_stride,
   });
 }
 
-template<typename traits, typename res_t>
-static void set_result(const int index, const res_t result, const TensorIteratorBase &iter, const int num_outputs) {
-  // static_assert(std::is_same<res_t, typename traits::arg2_t>::value, "data types must match");
+template <typename res_t>
+static void set_result(
```

Comment:
Is this re-formatting done to appease the linter, or is it something your adding? It muddies the diff and subsequent git blame, so I'd prefer you didn't.

Reply:
Yes, I re-formatted the code, and I reverted it now.